### PR TITLE
adding Dockerfile & build instructions

### PR DESCRIPTION
**Pull request by rAm1n**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+FROM colmap/colmap:latest
+
+
+MAINTAINER Paul-Edouard Sarlin
+
+RUN apt-get update -y
+RUN apt-get install python3 python3-pip unzip wget -y
+#RUN pip3 install -U pip
```
Comment (Skydes) [inline Dockerfile:9]: Clean up: remove the commented line and unnecessary line breaks

File: README.md [hunk 2]
```diff
@@ -26,6 +26,25 @@ pip install git+https://github.com/mihaidusmanu/pycolmap
 
 This codebase includes external local features as git submodules – don't forget to pull submodules with `git submodule update --init --recursive`. Your local features are based on TensorFlow? No problem! See [below](#using-your-own-local-features-or-matcher) for the steps.
 
+
+#### Docker setup
+
+You can also use the docker image provided that already has colmap and all requirements installed. After building the image, you would just need to setup jupyter and run it through browser:
```
Comment (Skydes) [inline README.md:32]:
```suggestion
We also provide a Docker image that includes COLMAP and other dependencies:
```

File: README.md [hunk 3]
```diff
@@ -26,6 +26,25 @@ pip install git+https://github.com/mihaidusmanu/pycolmap
...
+docker build -t hloc:latest .
+
+#run a container instance:
+
+docker run -it -p 8888:8888 hocl:latest
+
+#and start jupyter
+
+jupyter notebook --ip 0.0.0.0 --port 8888 --no-browser --allow-root
+```
```
Comment (Skydes) [inline README.md:44]: I suggest to remove comments and line breaks to make it compact - it shouldn't take much space in the README

**Comment by Skydes** [comment]
That's a great idea, thanks! The Dockerfile looks good. I have added some comments to improve the instructions.

Re downloading Aachen in the notebook: we can discuss in a separate PR. I am not against it, but the proposed changes only download the additional v1.1 images, not the full dataset. Currently only Aachen v1.0 is supported out of the box.

**Comment by rAm1n** [comment]
I applied the requested changes. Thanks for the feedback and for sharing this cool work! :)

File: README.md [hunk 4]
```diff
@@ -26,6 +26,18 @@ pip install git+https://github.com/mihaidusmanu/pycolmap
...
 This codebase includes external local features as git submodules – don't forget to pull submodules with `git submodule update --init --recursive`. Your local features are based on TensorFlow? No problem! See [below](#using-your-own-local-features-or-matcher) for the steps.
 
+We also provide a Docker image that includes COLMAP and other dependencies:
+
+```
+docker build -t hloc:latest .
+#run a container instance:
+docker run -it -p 8888:8888 hocl:latest
+#and start jupyter
+jupyter notebook --ip 0.0.0.0 --port 8888 --no-browser --allow-root
```
Comment (Skydes) [inline README.md:36]:
```suggestion
docker build -t hloc:latest .
docker run -it -p 8888:8888 hocl:latest
jupyter notebook --ip 0.0.0.0 --port 8888 --no-browser --allow-root
```

**Comment by Skydes** [comment]
Thanks a lot for the changes 👍

**Pull request merged**